src/tests/Microsoft.PowerFx.Interpreter.Tests/MutationScripts/Remove.txt

```diff
@@ -0,0 +1,14 @@
+﻿>> Set(list, Table({ Name: "One", ID: 1}, { Name: "Two", ID: 2}))
+Table({ID:1,Name:"One"},{ID:2,Name:"Two"})
+
+>> Set(list2, Table(First(list)))
+Table({ID:1,Name:"One"})
+
+>> ForAll(list2, Remove(list, ThisRecord))
+Table(Blank())
```

Comment:
>able(Blank()) [](http://example.com/codeflow?start=1&length=13)

We should add this test, to show that `list` was properly updated:

    >> list
    Table({ID:2,Name:"Two"})